microcom.c

```diff
@@ -183,6 +183,10 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	if (optind < argc) {
+		main_usage(1, "", "");
+	}
```

Comment:
I would drop the curly braces, otherwise looks good. I was annoyed by microcom behaving unexpected for positional parameters in the past, but procrastinated creating a patch for it. Thanks!